.github/workflows/pack-android-sdks.yml

```diff
@@ -0,0 +1,93 @@
+name: Package Android SDKs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+    - name: Build
+      run: |
+        ./scripts/package/package_android_sdk.sh linux
+        eval "$(python ./build_tools/set_sdk_vars.py ANDROID_TARGET_API_LEVEL ANDROID_BUILD_TOOLS_VERSION)"
+        echo "ANDROID_TARGET_API_LEVEL=$ANDROID_TARGET_API_LEVEL" >> $GITHUB_ENV
+        echo "ANDROID_BUILD_TOOLS_VERSION=$ANDROID_BUILD_TOOLS_VERSION" >> $GITHUB_ENV
+
+    - name: Archive results
```

Comment:
Is it worth to implement direct upload to S3 instead of github artifacts?

Reply:
nah, it's pretty rare action and I don't see it as a problem to upload it myself.